src/Core/System/Resources/config/listing.xml

```diff
@@ -36,5 +36,14 @@
             <helpText>Products are marked as new, if their release date does not date back past the set number of days.</helpText>
             <helpText lang="de-DE">Produkte werden als neu markiert, wenn ihr Erscheinungsdatum nicht länger als die angegebene Anzahl an Tagen zurückliegt.</helpText>
         </input-field>
+
+        <input-field>
+            <name>productCountSteps</name>
+            <label>Products per page</label>
+            <label lang="de-DE">Produkte pro Seite</label>
+            <helpText>Integers starting from 1 separated by comma</helpText>
+            <helpText>Ganzzahlen ab 1 getrennt mit Komma</helpText>
```

Comment:
Lang attribute missing 🤔

Reply:
Nope. en-GB is default.

Reply:
OOOOH the helpText lang attribute
Gotta fix fast

Thanks for pointing